pkg/addon/configpolicy/manifests/managedclusterchart/templates/deployment.yaml

```
@@ -111,6 +112,12 @@ spec:
           failureThreshold: 30
           periodSeconds: 10
         {{- end }}
+        {{- if and .Values.prometheus.enabled (eq .Values.kubernetesDistribution "OpenShift") }}
+        ports:
+        - name: metrics
+          protocol: TCP
+          containerPort: 8443
+        {{- end }}
         {{- if and .Values.prometheus.enabled (ne .Values.kubernetesDistribution "OpenShift") }}
```

Having an `else if` would be cleaner.
```suggestion
        {{- else if .Values.prometheus.enabled }}
```